operators/multiclusterobservability/controllers/multiclusterobservability/analytics/rightsizing_controller.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+// Licensed under the Apache License 2.0
+
+package analytics
+
+import (
+	"context"
+
+	"github.com/cloudflare/cfssl/log"
```

Comment:
What is the reason for using this log? It looks like other parts of MCO uses `sigs.k8s.io/controller-runtime/pkg/log`

Reply:
Updated the logs.